ops/prod/api.tf

```diff
@@ -59,3 +59,38 @@ module "simple_report_api" {
     # SPRING_JPA_PROPERTIES_HIBERNATE_DEFAULT_SCHEMA = "public"
   }
 }
+
+resource "azurerm_monitor_autoscale_setting" "prod_autoscale" {
+  name                = "SimpleReport Autoscaling"
+  resource_group_name = data.azurerm_resource_group.rg.name
+  location            = data.azurerm_resource_group.rg.location
+  target_resource_id  = module.simple_report_api.app_service_plan_id
+  profile {
+    name = "Peak Hours"
```

Comment:
the current name in azure is `Peak hours` slightly different from `Peak Hours`

Reply:
also the autoscale setting name in azure is
`prime-simple-report-appserviceplan-prod-Autoscale-485`
is the plan to just remove the old setting manually? or is this a representation of the existing setting in tf?

Reply:
Thank you for bearing with my delayed response, with the adjusted priorities!

There are some changes between what is in code and what is currently reflected in Azure. Just before this is rolled out, I will run a `terraform import`. That will pull the existing resource into the state file, and associate it with the code I have written here. Then, once this is merged, the existing autoscale settings will be updated with what is written in this file. We don't have to worry about discrepancies, as TF will automatically change what is reflected to match with what is in code.

Just to be safe, I will pick a night where we have an off-hour period to roll this out, since it will require that manual `terraform import` run before taking effect.

Reply:
I don't think a `terraform import` is even necessary since it's just a resource representing a particular group of settings. All you'd have to do is remove the existing autoscale settings and then merge this in.